test/Inputs/clang-importer-sdk/usr/include/ctypes.h

```diff
@@ -7,10 +7,15 @@ enum Tribool {
   True, False, Indeterminate
 };
 
-enum {
+enum
+#if defined(_WIN32)
+: unsigned long long
+#endif
```

Comment:
Can you leave a comment about why it's okay to have the explicit type here? (specifically, that we're not testing the inferred type on Windows because it's known to be broken)

Reply:
:-/ I think it's important to call out that it's not just *different* but not something we consider worth testing because it's behavior we don't like. Alternately, we should test it like we do the other anonymous one.

Reply:
Is it really behaviour that we don't like?  It is different, but, that's the beauty of ABI - you get what you get and you like it!  This is not something in the standard so the behaviour of the `enum` is correct either way.  What do you have in mind of testing the behaviour?

Reply:
Hm, fair point. According to the standard, "The choice of type is implementation-defined, but shall be capable of representing the values of all the members of the enumeration" (C11 6.7.2.2p4), so Int32 is *not* an okay choice…but it also says "The identifiers in an enumerator list are declared as constants that have type `int` and may appear wherever such are permitted." (C11 6.7.2.2p3), and as an anonymous enum that means you'd never actually be able to use this value as written. So both MSVC and GCC are bending the rules here.

I think that means we should test both behaviors.

Reply:
Attempting this in #23653.